stage3-py.sh

```diff
@@ -22,20 +22,18 @@ rubin_env_ver=$(mamba list rubin-env$ --no-banner --json \
 # Do the rest of the installation; pin moto to overcome jsonschema version
 # breakage.
 mamba install --no-banner -y \
-      "rubin-env-rsp==${rubin_env_ver}" \
-      "moto<4"
+      "rubin-env-rsp==${rubin_env_ver}"
 # Next, things on conda-forge not yet rolled into rubin-rsp-env
 mamba install --no-banner -y \
-      jupyterlab-variableinspector \
       gh
 # These are the things that are not available on conda-forge.
 # Note that we are not installing with `--upgrade`.  That is so that if
 # lower-level layers have already installed the package (e.g. T&S may have
 # already installed lsst-efd-client), pinned to a version they need, we won't
 # upgrade it.  But if it isn't already installed, we'll just take the latest
-# available.
-pip install \
-      socketio-client \
```

Comment:
Do we really not need socketio-client?

Reply:
I was going to add an equivalent from conda-forge, but it looks like there is no drop-in replacement.  On the other hand, it also doesn't look like it's used.